Heakhafaw_1105.xml

```diff
@@ -0,0 +1,136 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Sector xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" Tags="OTU" Abbreviation="Heak">
+  <Name Lang="as">Heakhafaw</Name>
+  <Credits><![CDATA[
+
+             <b>Heakhafaw</b> dot map was created by 
+             Peter G. Celella and James Holden and 	
+             appears in <cite>The MegaTraveller Alien
+             , Volume 2: Solomani &amp; Aslan</cite> 
+             (DGP, 1991).
+
+             Heakhafaw was refined and updated to  
+             T5 standards by Leigh Heron.
+
+    ]]></Credits>
+  <X>-6</X>
+  <Y>4</Y>
+  <DataFile Source="Solomani and Aslan" Milieu="M1105" Ref="https://travellermap.com/data/heakhafaw" />

```

Comment:
Don't include DataFile elements in the metadata file; they're only needed in the index (e.g. M1105.xml)